client/css/style.css

```diff
@@ -769,12 +771,20 @@ kbd {
 	font-size: 14px;
 	margin: 2px 0;
 	margin-bottom: 10px;
-	outline: 0;
 	padding: 8px 10px;
-	transition: border-color 0.2s;
+	transition: border-color 0.2s, box-shadow 0.2s;
 	width: 100%;
 }
 
+#chat .search:focus,
+.input:focus {
+	border-color: #84ce88;
+}
+
+#windows .input:hover {
```

Comment:
You can combine this with the one above.

Reply:
I'll also remove the `#window` in front of it.

Reply:
Done.